### PR TITLE
Update com.fasterxml.jackson.core Version

### PR DESCRIPTION
**Pull request by Akanksha-kedia**
image

### CODE REVIEW

File: presto-hudi/pom.xml [hunk 1]
```diff
@@ -212,7 +212,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-annotations</artifactId>
-            <version>2.10.2</version>
+            <version>2.13.2</version>
```
Comment (7c00) [inline presto-hudi/pom.xml:215]: Let's use `{dep.jackson.version}` instead.

File: presto-iceberg/pom.xml [hunk 2]
```diff
@@ -526,6 +532,7 @@
                 <plugin>
                     <groupId>org.basepom.maven</groupId>
                     <artifactId>duplicate-finder-maven-plugin</artifactId>
+                    <version>1.5.1</version>
```
Comment (7c00) [inline presto-iceberg/pom.xml:535]: it's unnecessary to specify the version as it is managed in base pom (airbase)

**Review by 7c00 — CHANGES_REQUESTED** [review]

**Comment by 7c00** [comment]
@Akanksha-kedia Thanks for your contribution. I leave some comments. And please follow [Pull Request Checklist](https://github.com/prestodb/presto/blob/master/CONTRIBUTING.md#pull-request-checklist) to update the PR description.

File: pom.xml [hunk 3]
```diff
@@ -143,7 +144,6 @@
         <module>presto-cli</module>
         <module>presto-benchmark-driver</module>
         <module>presto-server</module>
-        <module>presto-docs</module>
```
Comment (jaystarshot) [inline pom.xml:146]: I don't think this is intended

**Comment by Akanksha-kedia** [comment]
Will check

**Review by jaystarshot — COMMENTED** [review]
You will need to update the description of the PR as mentioned above with the template

**Pull request closed without merging**